mandatory/2-function-creation.js

```diff
@@ -15,26 +17,56 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200) {
+return totalPrice * 0.9;
+} else {
+return totalPrice * 0.95;
+};
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let number = 1;
+while (number <= limit) {
+if (number % 2 == 1) {
+console.log(number);
+}
+number++;
+};
+}
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
+function buyTwoGetTheCheapestFree(price1, price2) {
+  if (price1 > price2) {
+return price1;
+} else if (price2 > price1) {
+return price2;
+} else {
+  return price1||price2;
```

Comment:
If the price is the same then you can simply return whicever value

Reply:
true! I thought this way i'd be showing that i'm aware there's no difference. but i guess code length  and simplicity matters more.. edited now  👍